crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -654,14 +571,14 @@ async fn get_validation_package_remote(
                 }
 
                 // Fallback to gossiper if author is unavailable
-                if let Some(from_agent) = from_agent {
-                    if let Some(validation_package) = cascade
-                        .get_validation_package(from_agent, header_hashed)
-                        .await?
-                    {
-                        return Ok(Some(validation_package));
-                    }
-                }
+                // if let Some(from_agent) = from_agent {
+                //     if let Some(validation_package) = cascade
+                //         .get_validation_package(from_agent, header_hashed)
+                //         .await?
+                //     {
+                //         return Ok(Some(validation_package));
+                //     }
+                // }
```

Comment:
needed?

Reply:
These will need to be re-implemented once we figure out what to do with validation packages.